app/routes/screens/clubFeed/styles.js

```diff
@@ -8,9 +8,9 @@ export default StyleSheet.create({
   },
   scroll: {
     marginTop: 20,
-    marginBottom: 20,
-    marginLeft: 10,
-    marginRight: 10
+    marginBottom: 1,
+    marginLeft: 1,
+    marginRight: 1
```

Comment:
I ve planned to removed react-native-elements dependency soon. 
Style will be highly impacted then. So, no need to waste time now with that.